content/develop/data-types/probabilistic/bloom-filter.md

```diff
@@ -10,16 +10,13 @@ categories:
 - kubernetes
 - clients
 description: Bloom filters are a probabilistic data structure that checks for presence
```

Comment:
All the probability data types have a similar description. It would be awkward to modify only the Bloom filter description. Maybe we can do this in a separate ticket?